blocks/shared-styles/_children/promos/extra-large/index.jsx

```diff
@@ -133,7 +133,7 @@ const ExtraLargePromoPresentation = ({
 
 ExtraLargePromoPresentation.propTypes = {
   content: PropTypes.object,
-  showOverline: PropTypes.string,
+  showOverline: PropTypes.bool,
```

Comment:
new? possibly better in another pr?

Reply:
figured it was a small enough change.  Noticed the warning in the console and it's supposed to be a boolean (this is the only one that is not and I think it snuck through in the refactor.  If you think we need another PR, I can revert that.
```suggestion
  showOverline: PropTypes.string,
```

Reply:
no worries it's fine